src/common/errors/errors.ts

```diff
@@ -5,7 +5,14 @@ export class NotFoundError extends Error {
   }
 }
 
-export class ValidationError extends Error {
+export class ConflictError extends Error {
+  constructor(...args: ConstructorParameters<typeof Error>) {
+    super(...args)
+    this.name = this.constructor.name
+  }
+}
+
+export class BadRequestError extends Error {
```

Comment:
Previous naming was more suitable and reflected business logic better.
```suggestion
export class ValidationError extends Error {
```

Reply:
Agreed, fixed